src/main/java/bio/terra/stairway/HookWrapper.java

```diff
@@ -0,0 +1,70 @@
+package bio.terra.stairway;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HookWrapper {
+  private static final Logger logger = LoggerFactory.getLogger(HookWrapper.class);
+  private Stairway stairway;
+
+  public HookWrapper(Stairway stairway) {
+    this.stairway = stairway;
+  }
+
+  public HookAction startFlight(FlightContext context) {
+    return handleHook(context, HookType.startFlight);
```

Comment:
I'm not a fan of this pattern the takes the specific endpoint, makes it general, only to switch to make it specific again.
I'd probably have just cut'n'pasted. The cool kids pass functions as parameters. That might work here.

Reply:
I opted for the cut'n'pasted. I was attempting to avoid duplicated code, but this is also a fair point!